src/main/java/com/minecolonies/client/gui/WindowTownHall.java

```diff
@@ -253,11 +399,29 @@ public void onOpened()
         super.onOpened();
         int citizensSize = townHall.getColony().getCitizens().size();
 
-        //TODO - Base these on server-side computed statistics
         int workers     = 0;
         int builders    = 0;
         int deliverymen = 0;
 
+        for(CitizenDataView citizen: citizens)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'citizen' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)